hash_practice/exercises.py

```diff
@@ -1,19 +1,52 @@
 
+def create_anagram_key(string):
+    return ''.join(sorted(string))
+
+
 def grouped_anagrams(strings):
     """ This method will return an array of arrays.
         Each subarray will have strings which are anagrams of each other
-        Time Complexity: ?
-        Space Complexity: ?
+        Time Complexity: O(n*m)
+        Space Complexity: O(n)
     """
-    pass
+    grouped_words = {}
+    anagrams = []
+
+    for ele in strings:
+        if grouped_words.get(create_anagram_key(ele)) == None:

```

Comment:
Just a minor efficiency thing, if you find yourself calling the same function multiple times with the exact same arguments like you do with `create_anagram_key(ele)` here, it's usually worth saving the result into a local variable, since every call will recalculate the same data that doesn't change.